com.github.neil_morrison44.pocket-sync.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.github.neil_morrison44.pocket-sync
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
43 is out.

Reply:
I can't use 43 otherwise I would have to compile `webkit2gtk-4.0` and `javascriptcoregtk-4.0` myself, which the Rust bindings used by [tauri](https://github.com/tauri-apps/tauri/) doesn't support yet.